### PR TITLE
updated install link and instructions for R-Pi

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Removed old demo which is no longer needed

### CODE REVIEW

**Review by adityapatwardhan — APPROVED** [review]
LGTM

File: docs/installation/linux.md [hunk 1]
```diff
@@ -672,6 +672,35 @@ chmod a+x powershell-6.0.0-rc-x86_64.AppImage
...
 
+## Raspbian
+
+Currently, PowerShell is only supported on Rasbian Stretch.
+
+### Installation
+
+```sh
+# Install prerequisites
+sudo apt-get install libunwind8 libicu57
```
Comment (anmenaga) [inline docs/installation/linux.md]: In my test, to run PS-RC on Rasbian Stretch, I didn't have to deploy libicu57; only libunwind8.

**Review by anmenaga — APPROVED** [review]
LGTM

File: docs/installation/linux.md (hunk 1 above)
Comment (SteveL-MSFT) [reply docs/installation/linux.md]: Ok, I'll remove that.

**Review by TylerLeonhardt — APPROVED** [review]
LGTM

**Comment by adityapatwardhan** [comment]
Restarted AppVeyor build.

**Comment by msftgits** [comment]
Closing & reopening to bump status checks on CLA (this is a test, thanks for your patience)

**Comment by jeffwilcox** [comment]
FYI @SteveL-MSFT the CLA part is validated now

**Pull request merged**